libs/api/domains/national-registry-x-road/src/models/nationalRegistryChildGuardianship.model.ts

```diff
@@ -0,0 +1,13 @@
+import { Field, ObjectType, ID } from '@nestjs/graphql'
+
+@ObjectType()
+export class NationalRegistryChildGuardianship {
```

Comment:
This name might clash with the national-registry domain, since the `NationalRegistry` prefix is bound to that domain.

Reply:
The other models in the `national-registry-x-road` domain use the same naming though, should it rather be `NationalRegistryXRoad`?

Reply:
yup